StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -53,6 +53,10 @@ class PaymentSheetLinkAccount: PaymentSheetLinkAccountInfoProtocol {
     let apiClient: STPAPIClient
     let cookieStore: LinkCookieStore
 
+    let useModernMobileEndpoints: Bool
```

Comment:
I don't have much context here, but "modern" MobileEndpoints feels weird.  Feels like we'll look back on this in a year and think that it isn't so modern anymore.

Reply:
Ideally we'd delete the old ones someday! I also considered something like `useMobileSpecificEndpoints`, but I'm not sure if that's much clearer. `useAttestationEndpoints`?